README.md

```diff
@@ -519,3 +519,176 @@ You can omit this argument to ignore this filter.
 
 This is the `reference` you gave at the time of sending the notification. The `reference` can be a unique identifier for the notification or an identifier for a batch of notifications.
 
+## Get a template by ID
+
+```
+response = notifications_client.get_template(
+    'template_id'
+)
```

Comment:
we should clarify in this section that this'll return the most recent version of the template only